### ISSUE TITLE
Failing test: test_subprocess_manager_invalid_when_exe_missing

### ISSUE THREAD

**Issue by jaraco** [jaraco/jaraco.vcs#26]
Originally reported by: **Marc Abramowitz (BitBucket: [msabramo](http://bitbucket.org/msabramo), GitHub: [msabramo](http://github.com/msabramo))**

---

```
$ py.test --tb=short hgtools
Test session starts (platform: darwin, Python 2.7.6, pytest 2.5.2, pytest-sugar 0.3.4)
plugins: cov, sugar
...
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_subprocess_manager_invalid_when_exe_missing ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
hgtools/tests/test_git.py:17: in test_subprocess_manager_invalid_when_exe_missing
    assert not mgr.is_valid()
hgtools/managers/cmd.py:143: in is_valid
    return super(Command, self).is_valid() and self.version_suitable()
hgtools/managers/cmd.py:147: in version_suitable
    act_ver = pkg_resources.parse_version(self.version())
hgtools/managers/cmd.py:187: in version
    lines = iter(self._invoke('version').splitlines())
hgtools/managers/subprocess.py:19: in _invoke
    stderr=subprocess.PIPE, cwd=self.location, env=self.env)
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py:709: in __init__
    errread, errwrite)
/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py:1326: in _execute_child
    raise child_exception
E   OSError: [Errno 2] No such file or directory
...
Results (13.91s):
   35 passed
   1 failed
      - /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py:1326: OSError: [Errno 2] No such file or directory
```

---
- Bitbucket: https://bitbucket.org/jaraco/hgtools/issue/26

**Comment by jaraco** [comment]
_Original comment by_: **Marc Abramowitz (BitBucket: [msabramo](http://bitbucket.org/msabramo), GitHub: [msabramo](http://github.com/msabramo))**

---

See https://bitbucket.org/jaraco/hgtools/pull-request/5/fix-issue-26-failing-test/diff for proposed fix.

**Comment by jaraco** [comment]
_Original comment by_: **Jason R. Coombs (BitBucket: [jaraco](http://bitbucket.org/jaraco), GitHub: [jaraco](http://github.com/jaraco))**

---

Correct usage of super to capture missing executable. Fixes #26
